source/wp-content/themes/wporg-wasm/src/wasm-demo/src/style.scss

```diff
@@ -148,39 +159,87 @@
 			border: 1px solid #dedede;
 			border-radius: 4px;
 			padding: 8px 12px;
-
-			a {
-				text-decoration: none;
-			}
+			flex-grow: 1;
 
 			&.is-active {
-				border-color: var(--wp--preset--color--vivid-cyan-blue);
+				border-color: var(--wp--preset--color--charcoal-1);
+			}
+			&:hover {
+				border-color: var(--wp--preset--color--blueberry-1);
 			}
 		}
 
 		.wporg-tab-item-list__item-name {
 			margin: auto;
 		}
+
+		.components-checkbox-control__input-container {
+			margin-right: 0;
+		}
+		.components-base-control__field {
+			margin-bottom: 0;
+		}
 	}
 }
 
+html {
+	margin-top: 0 !important;
+}
+
+#wpadminbar {
+	display: none !important;
+}
```

Comment:
I assumed this css file is only loaded on a single page. Is this a correct assumption @StevenDufresne, or will this break other pages ?

Reply:
Yeah, Gutenberg is responsible for loading this. If I remember correctly, I'm pretty sure it checks for the existence of the block first.